frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -180,6 +183,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  {
+    type: 'OCSKebabActions',
+    properties: {
+      kind: PersistentVolumeClaimModel.kind,
+      label: 'Clone',
+      apiGroup: models.OCSServiceModel.apiGroup,
+      callback: (kind, resource) => () => {
+        const clusterObject = { resource };
+        import(
+          './components/modals/clone-pvc-modal/clone-pvc-modal' /* webpackChunkName: "ceph-storage-clone-pvc-modal" */
+        )
+          .then((m) => m.default(clusterObject))
+          .catch((e) => {
+            throw e;
+          });
+      },
+    },
+  },
```

Comment:
There is one `KebabActions` extension. What is the reason for not using that ?

Reply:
I agree we don't need to create a new extension.